utils/mamUtils.js

```diff
@@ -287,9 +287,9 @@ class mam {
 
   /* temporary workaround */
   async createIdentity(packet) {
-    packet = deepCopy(packet);
-    delete packet.sk;
-    await Cert.attach(packet, packet.uuid, 'I');
+    const packetCopied = deepCopy(packet);
+    delete packetCopied.sk;
```

Comment:
Why "delete" here? Can you copy certain field instead?

Reply:
The "delete" operator allows us to remove a property from the object.
We can copy the certain fields, but the code will become long and hard to read.

Because this function is a "temporary workaround", I'll remove this method in the
incoming PR.

Reply:
Good. Make it elegant and self-explanatory. I will wait for another pull request which removes such workaround first and then merge this PR.